libs/aws/langchain_aws/chat_models/bedrock_converse.py

```diff
@@ -307,8 +308,49 @@ class Joke(BaseModel):
     
     Profile should either have access keys or role information specified.
     If not specified, the default credential profile or, if on an EC2 instance,
-    credentials from IMDS will be used. See: 
-    https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html
+    credentials from IMDS will be used. 
+    See: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html
+    """
+
+    aws_access_key_id: Optional[SecretStr] = Field(
+        default_factory=secret_from_env("AWS_ACCESS_KEY_ID", default=None)
+    )
+    """AWS access key id. 
+    
+    If provided, aws_secret_access_key must also be provided.
+    If not specified, the default credential profile or, if on an EC2 instance,
+    credentials from IMDS will be used.
+    See: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html
+    
+    Will be read from env var 'AWS_ACCESS_KEY_ID' if it is set and env var 
+    aws_access_key_id init arg is not passed in.
```

Comment:
Can we simplify these doc statements. Maybe something like:

If not provided, will be read from 'AWS_ACCESS_KEY_ID' environment variable.